face_detection.mjs

```diff
@@ -0,0 +1,77 @@
+import { resize } from "./preprocess.mjs";
+import { getOrt } from "./onnxruntime.mjs";
+import { tensorToMatrix } from "./inference_utils.mjs";
+
+export class GuruFaceDetector {
```

Comment:
```suggestion
export class UltrafaceFaceDetector {
```

The naming convention we've been using includes the name of the model (e.g., `MixFormerObjectTracker`, `OCSortTracker`, `YOLOXDetector`, etc). `GuruPoseEstimator` is an exception only because we don't really have a name for our model, but it is ours.

Reply:
Resolved in latest commit.